net/developer-guide/presentation-content/powerpoint-shapes/shape-types/custom-shapes/_index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Custom Shape
+type: docs
+weight: 10
+url: /net/custom-shape/
+---
+
+# Customization of the Shape Geometry (Shape Points Editing)
+  * [Add Custom Points to the Shape](#add-custom-points-to-the-shape-)

```

Comment:
The navigation link is not working
Please change  #add-custom-points-to-the-shape- to #add-custom-points-to-the-shape